packages/big-design/src/components/Worksheet/editors/ModalEditor/ModalEditor.tsx

```diff
@@ -36,13 +36,15 @@ const InternalModalEditor = <T extends WorksheetItem>({ cell, formatting, isEdit
   return (
     <Flex justifyContent="space-between" alignItems="center" flexWrap="wrap">
       <StyledFlexItem paddingRight="small" flexShrink={1}>
-        <Small color="secondary70" ellipsis title={renderedValue}>
+        <Small color={cell.disabled ? 'secondary50' : 'secondary70'} ellipsis title={renderedValue}>
           {renderedValue}
         </Small>
       </StyledFlexItem>
-      <StyledButton onClick={handleClick} ref={buttonRef} variant="subtle">
-        Edit
-      </StyledButton>
+      {!cell.disabled && (
```

Comment:
🍹 Should we completely not render the edit button when cell is disabled, or render it as disabled as well as the cell?

Reply:
Good question. This was definitely something I decided, since to me it doesn't make sense to display the `Edit` button if a cell is disabled and it can't be edited. Do you think it would be a better idea to show it?

Reply:
Idk, I'm torn. I think it's fine without for right now, just might be good to have design input at some point.

Reply:
I decided to add it in and we can remove later if needed.